frontend/src/scenes/data-management/revenue/RevenueEventsSettings.tsx

```diff
@@ -34,11 +37,17 @@ export function RevenueEventsSettings(): JSX.Element {
         [updatePropertyName]
     )
 
+    const buttonRef = useRef<HTMLButtonElement | null>(null)
+
     return (
         <div className="space-y-4">
-            <div>
-                <p>Add revenue tracking for Custom events in Web analytics</p>
-            </div>
+            <ProductIntroduction
+                productName="Revenue tracking"
+                thingName="revenue event"
+                description="Revenue events are used to track revenue in Web analytics. You can choose which custom events PostHog should consider as revenue events, and which event property corresponds to the value of the event."
+                isEmpty={events.length === 0}
+                action={() => buttonRef.current?.click()}
```

Comment:
What a lovely hack :)

Reply:
It actually feels pretty nice in the UI as well - the taxonomic popover opens upwards so even if you click the ProductIntro's button, your mouse will be over the list of events